### ISSUE TITLE
Jetpack Connect: Language Adjustments

### ISSUE THREAD

**Issue by ashleighaxios** [Automattic/wp-calypso#14553]
There are currently a few Jetpack connection screens per plan level, largely accessed via [this page](https://jetpack.com/pricing), where the language can be confusing to customers since we can’t tell if a user has Jetpack installed or not at this point, we don’t know who they are, or what their site address is. For now, this issue aims to make these screens more apt for the variety of customer uses by making the language appropriate for each use case and customer flow.

_This is related to p6TEKc-1cr by @annezazu. As noted there, we can later embark on a larger quest to create a ‘purchase-first’ flow for the urls linked below. This issue however, calls for a much more immediate fix to the confusion of the existing screens._

### Free
**Currently:**
[Link to it](https://wordpress.com/jetpack/connect)
<img width="944" alt="01_free" src="https://cloud.githubusercontent.com/assets/22752103/26555313/7a675d90-4462-11e7-88f1-1596391d9b32.png">
^

> **Connect a self-hosted WordPress**
We'll be installing the Jetpack plugin so WordPress.com can connect to your self-hosted WordPress site.

**Proposed:**

> **Get Jetpack**
You're on your way to being backed up and secure, please share your site URL.

### Personal
**Currently:**
[Link to it](https://wordpress.com/jetpack/connect/personal)
<img width="944" alt="02_personal" src="https://cloud.githubusercontent.com/assets/22752103/26555315/7c124af6-4462-11e7-994a-0e9e2f7cdba6.png">
^

> **Get Jetpack Personal**
To start securing and backing up your site, first install Jetpack, then purchase and activate your plan.

**Proposed:**

> **Get Jetpack Personal**
You're on your way to being backed up and secure, please share your site URL.

### Premium
**Currently:**
[Link to it](https://wordpress.com/jetpack/connect/premium)
<img width="944" alt="03_premium" src="https://cloud.githubusercontent.com/assets/22752103/26555319/7e876ee2-4462-11e7-93f2-47cd9df8afeb.png">
^

> **Get Jetpack Premium**
To start securing and backing up your site, first install Jetpack, then purchase and activate your plan.

**Proposed:**

> **Get Jetpack Premium**
You're on your way to being backed up and secure, please share your site URL.

### Professional
**Currently:**
[Link to it](https://wordpress.com/jetpack/connect/pro)
<img width="944" alt="04_professional" src="https://cloud.githubusercontent.com/assets/22752103/26555323/802748e4-4462-11e7-900d-d59b88f30af8.png">
^

> **Get Jetpack Professional**
To start securing and backing up your site, first install Jetpack, then purchase and activate your plan.

**Proposed:**

> **Get Jetpack Professional**
You're on your way to being backed up and secure, please share your site URL.

Cc @rickybanister

**Comment by rickybanister** [comment]
These are nice improvements. I think the message for free users needs some work because backups are not part of the Free plan.

Perhaps @richardmuscat can give these a read-through before we proceed.

**Comment by benhuberman** [comment]
Hey @ashleighaxios -- this thread fell through the cracks despite the needs copy review label; apologies. I'll take a look and propose edits, if any, tonight (PST).

**Comment by benhuberman** [comment]
@ashleighaxios -- Having taken a look at all the screens above, it sounds like what you're looking for here is a message that's generic enough to apply to all use cases, but compelling enough to push people to share their URL. Is that more or less correct? 

In case it is, how about the following:

> **Get Jetpack [Plan name, where applicable]**
>  Your Jetpack-powered site is about to take off. To get started, enter your site's URL below.

Beyond the point @rickybanister made about "backed up" not being applicable for free users, I also wasn't a huge fan of the passive voice in this context. Happy to try alternatives or tweak if you don't think it's quite there yet.

**Comment by ashleighaxios** [comment]
Thanks @benhuberman, I think this will help and agree on active voice being better here. 

@annezazu, since you brought this to our attention, want to give it a quick review?

**Comment by annezazu** [comment]
Something is still off about this. I still don't feel it's compelling or descriptive enough. My site is about to take off? What? I also could see this being confusing for folks who aren't native English speakers. It sounds like we might crash their site heh. I also think for some reason "Jetpack-powered site" doesn't make much since. 

In the spirit of "bring solutions not just problems", some options:

> Get Jetpack [Plan name, where applicable]
> Your Jetpack features need to be unlocked. To get started, enter your site's URL below.

Or

> Get Jetpack [Plan name, where applicable]
> Unlock all of Jetpack's features. To get started, enter your site's URL below.

Or something super simple (want a plan, get started by doing x):

> Get Jetpack [Plan name, where applicable]
> To get started, enter your site's URL below.

Or

> Get Jetpack [Plan name, where applicable]
> Security and backups are clicks away. To get started, enter your site's URL below. 

Or

> Get Jetpack [Plan name, where applicable]
> Security and backups are clicks away. To get started, enter your site's URL below. 

Or

> Finish purchasing Jetpack [Plan name, where applicable]
> You're a few steps away from a secure site. To start, enter your site's URL below. 

Or a more conversational style:

> Get Jetpack [Plan name, where applicable]
> We have a few more steps before we can backup and secure your site. To start, enter your site's URL below. 

^ Hopefully this helps things and not muddles them. This is what happens when I have too much coffee too early in the day ;)

**Comment by ashleighaxios** [comment]
Helpful - thanks, all! 

_Pinging @richardmuscat again just in case - though I believe he's still AFK._

I think we need to avoid saying `backups` as a new offering since we don't know which plan they may be upgrading from (IE: a paid plan to a higher level paid plan). So how about we simplify this way down so that all of them get:

> **Get Jetpack [Plan name, if above Free]**
> To get started, enter your site's URL below.

**Comment by rickybanister** [comment]
We can just be specific about the service tier. For instance `real-time backups are just clicks away`.

**Comment by ashleighaxios** [comment]
Let's go generic for now and then update later with tiers. Would be good to have Richard's feedback if we're sharing more specifics for each tier and, since this is after a plan selection for most, I don't think its as essential to spell out the selling point here.

**Comment by tyxla** [comment]
These could still be beneficial, given the latest steps we introduced (site type, site vertical, user type)
